docs/user_guide/Initial_Server_Configuration.rst

```diff
@@ -6,56 +6,66 @@ Initial SIMP Server Configuration
-intended to make the system initial configuration straightforward and
-repeatable.
+In these instructions, we will be using the ``config`` and ``bootstrap`` of the
+``simp`` command.  The ``simp`` command provides a CLI intended to make the
+initial configuration of the SIMP server straightforward and repeatable.
 
 .. NOTE::
 
    For a list of the commands ``simp`` provides, type ``simp help``. Type
-   ``simp help <Command>`` for more information on a specific command.
+   ``simp <Command> --help`` for more information on a specific command.
 
 Configuring the SIMP Server
 ---------------------------
 
 .. IMPORTANT::
 
    Correct time across all systems is important to the proper functioning of
-   SIMP and Puppet in general.
+   SIMP (and Puppet in general).
 
-   If a user has trouble connecting to the Puppet server and errors regarding
-   certificate validation appear, check the Puppet server and client times to
-   ensure they are synchronized.
+   If a Puppet agent receives errors regarding certificate validation while
+   connecting to the Puppet server, compare the time on the Puppet server and
+   agent and ensure they are synchronized.
 
 .. WARNING::
 
-   Keep in mind as the installation process begins that Puppet does not
-   work well with capital letters in host names. Therefore, they should
-   not be used.
+   Puppet has problems when hostnames contain capital letters
+   (`SERVER-1809`_) — do not use them!
 
-For the remainder of the document, we will assume that you use the ISO
-installation method and that you are logging in using a ``simp`` local user.
-Use the appropriate user for your environment if you installed via an alternate
-method.
+   .. _SERVER-1809: https://tickets.puppetlabs.com/browse/SERVER-1809
+
+.. NOTE:: The remainder of this section assumes that:
+
+   * You started by :ref:`gsg-installing_simp_from_an_iso`
+   * You have logged in using the ``simp`` local user account (created by
+     the ISO installation)
+
+   Use the appropriate user for your environment if you installed via an
+   alternate method.
 
 #. Log on as ``simp`` and run ``su -`` to gain root access.
-#. Type ``simp config`` and configure the system as prompted.
+#. Run ``simp config`` and configure the system as prompted.
 
    - ``simp config`` will prompt you for system settings and then apply them as
      appropriate for bootstrapping the system.
 
    - When applicable, ``simp config`` will present you with a recommendation
-     for each setting (variable).  To keep a recommended value, press *Enter*.
-     Otherwise, enter your desired value.
+     for each setting.  To keep a recommended value, press *Enter*.  Otherwise,
+     enter your desired value.
+
+   - ``simp config`` generates a log file in ``/root/.simp`` containing details
+     of the configuration selected and actions taken.
+
+  .. NOTE::
+
+      For more details about the installation variables set by ``simp config``
```

Comment:
The section on installation variables is confusing because the installations variables are tree dependent.  IMHO we should take that section out and simply encourage the user to use the --dry-run option and examine the generated file, instead.

Reply:
@lnemsick-simp Seems reasonable, do you want to change it to make it make sense to you?

Reply:
I'll make the change, but in a separate PR.

Reply:
> The section on installation variables (specifically the table) is confusing because the installations variables are tree dependent. IMHO we should take that section out and simply encourage the user to use the --dry-run option and examine the generated file, instead. The comments in the generated file are what was copied into the installation variables section.

@lnemsick-simp @trevor-vaughan I don't like the idea of removing the only mention most of these settings currently have in the docs.
* Users will reasonably expect to find something about them, at least by searching the SIMP documentation (within RTD, or just googling). 
  * Anecdotally: I just fielded a support request _this morning_ about an (undocumented in `simp-doc`)  `simp config` variable from a previous version of SIMP.
* It's useful to have this information in RTD's link-friendly format, which we can provide to users during support.

Reply:
There are only two sections under **Advanced Configuration**.  They are _both_ a bit confusing for the same reasons, so they should probably share the same fate:
* The **Installation Variables** section (which will be effectively empty, if it is replaced with an instruction to run `simp config --dry-run`).
* The **simp config Actions** section, which is just as unclear about the conditions under which the actions in each category will/won't be applied—currently, the best contextual clues are provided by the "Installation Variables" table.

If the current list seems too confusing, I'd prefer that we focus on making it more clear that the table items are a documented _superset_ of `simp config`'s available "variables,"  and that arbitrary items in an answers file may not be honored or behave as expected, depending on `simp config`'s internal logic and the current progress of the questionnaire.

Reply:
OK.   We'll fix the Advanced Configuration section in a different ticket.